theme/errorList.less

```diff
@@ -21,4 +21,8 @@
         border-top: 1px #e5e5e5 solid;
         padding: 1em;
     }
-}
+
+    #noErrorsMessage {
+        height: 50px;
```

Comment:
Is this necessary? Instead would it be possible to adjust the line height or padding on the parent such that you get a good height when there's only one line of text?

Reply:
Actually, it is necessary since there's a fixed height for when there are errors

Reply:
also, preferable to use rem instead of px